RELEASE/scripts/autoscend/auto_restore.ash

```diff
@@ -677,7 +677,14 @@ __RestorationOptimization __calculate_objective_values(int hp_goal, int mp_goal,
 		}
 		else if(metadata.type == "skill")
 		{
-			available = floor(get_value("mp_starting") / mp_cost(to_skill(metadata.name)));
+			if(to_skill(metadata.name).dailylimit != -1)
+			{
+				available = to_skill(metadata.name).dailylimit;
```

Comment:
does this still need to be limited by available mp? i feel like there is (or could be) some limited heal that cost mp but idk.